### PR TITLE
Migrate GitHub action to conda-incubator/setup-miniconda

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

**Comment by ElDeveloper** [comment]
Well that's weird my changes to the CI script worked on the first try. I think I need to go buy a lottery ticket.  @antgonza @kwcantrell this is ready for you to review whenever you get a chance.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -32,42 +32,48 @@ jobs:
...
       - name: Install Emperor
+        shell: bash -l {0}
         run: |
-          conda create --yes -n tester python=${{ matrix.python-version }} pip numpy 'scipy>=0.17.0' matplotlib pandas flake8 pep8 jupyter coverage cython scikit-learn requests
-          conda install -n tester -c conda-forge phantomjs --yes
-          conda run -n tester pip install sphinx numpydoc sphinx-bootstrap-theme
-          conda run -n tester pip install -e ".[all]"
+          conda install pip numpy 'scipy>=0.17.0' matplotlib pandas flake8 pep8 jupyter coverage cython scikit-learn requests
+          conda install -c conda-forge phantomjs --yes
+          pip install 'sphinx<4' numpydoc sphinx-bootstrap-theme
```
Comment (antgonza) [inline .github/workflows/main.yml:57]: Should this be part of the setup.py? Either way, why <4?

**Review by antgonza — APPROVED** [review]
One question.

File: .github/workflows/main.yml (hunk 1 above)
Comment (ElDeveloper) [reply .github/workflows/main.yml]: Yes, good catch. I moved this whole line out of the recipe. This should all be handled by setup.py.

**Comment by antgonza** [comment]
Looks good, do we need another review or is this ready to be merged ?

**Comment by ElDeveloper** [comment]
Ready to be merged. :+1:

**Comment by ElDeveloper** [comment]
Thanks @antgonza.

**Pull request merged**